docs/api/migration-guides/qiskit-1.0.mdx

```diff
@@ -0,0 +1,594 @@
+---
+title: Qiskit 1.0 Migration Guide
+description: Adapt to changes in Qiskit 1.0
+---
+
+# Qiskit 1.0 Migration Guide
+
+
+## Packaging changes
+
+Qiskit 1.0 uses a different packaging structure to previous Qiskits, and may cause problems in environments using packages that are not ready for Qiskit 1.0.
+
+<Admonition type="caution">
+We **do not** recommend attempting to upgrade an existing Python virtual environment from Qiskit 0.45 or before to Qiskit 1.0 in place.
+</Admonition>
+
+
+### For users
+
+You must start a new virtual environment to install Qiskit 1.0.
+It is very tricky and error-prone to upgrade an existing installation in-place to Qiskit 1.0.
+
+<Admonition type="info">
+This section will use the `venv` module that is part of the Python standard library in its examples.
+If you use a different tool (such as `virtualenv` or `conda`), you should consult its documentation if you are unsure of how the commands line up.
+
+The examples will assume a `bash`-like shell for non-Python command-line inputs.
```

Comment:
I would also say "including on windows" for this too because that's the odd one out I think most mac and linux users will just assume this is the case.

Reply:
I switched to just using the (hopefully) correct PowerShell commands in 75f6e3a.